marl_eval/json_tools/pull_neptune_data.py

```diff
@@ -0,0 +1,71 @@
+# python3
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import zipfile
+from typing import List
+
+import neptune
+from colorama import Fore, Style
+from tqdm import tqdm
+
+
+def pull_neptune_data(
+    project_name: str,
+    tag: List,
+    store_directory: str = "./downloaded_json_data",
+    neptune_data_directory: str = "metrics",
+) -> None:
+    """Pulls the experiments data from Neptune to local directory.
+
+    Args:
+        project_name (str): Name of the Neptune project.
+        tag (List): List of tags.
+        store_directory (str, optional): Directory to store the data.
+        neptune_data_directory (str): Directory in Neptune where the data is stored.
+    """
+    # Get the run ids
+    project = neptune.init_project(project=project_name)
+    runs_table_df = project.fetch_runs_table(state="inactive", tag=tag).to_pandas()
```

Comment:
Did you try this with multiple tags? Or is it just neptune requires a list here instead of just a string?

Reply:
I just tested it and yes it is working if we use multiple tags, just something to mention these tags should all exist in the experiment else the experiment won't be taken into account.